README.md

```diff
@@ -1,17 +1,34 @@
-<h1 align="center">
-
-🌀 Portal.JS<br/>
-The javascript framework for<br/>
-data portals
-
+<h1 align="center">🌀 Portal.JS
+> The JavaScript framework for data portals
 </h1>
 
-🌀 `portal` is a framework for rapidly building rich data portal frontends using a modern frontend approach. `portal` can be used to present a single dataset or build a full-scale data catalog/portal.
```

Comment:
I would keep this intro paragraph *above* the ToC so people get an immediate sense.